### PR TITLE
07046 Fixed excessive logging caused by MemoryIndexDiskKeyValueStoreMergeHammerTest

### LINKED ISSUE #7046: MerkleDB Unit Tests Hang
Several times in the last week, I have observed the platform unit test suite get stuck and run for many hours. When I observe this, I cancel the test suite. Always, the workflow seem to be in the middle of the JasperDB tests.

Example: https://github.com/hashgraph/hedera-services/actions/runs/5224279727/jobs/9432293344?pr=7022

### LINKED ISSUE #7083: Hammer Test Log Spam
The test `MemoryIndexDiskKeyValueStoreMergeHammerTest` is generating a ton of log output, which is causing the `Publish Junit Test Report` job to fail.

```
Error: lxml.etree.XMLSyntaxError: CData section too big found, line 50459, column 6188
Error: Error processing result file: CData section too big found, line 50459, column 6188 (TEST-com.swirlds.merkledb.files.MemoryIndexDiskKeyValueStoreMergeHammerTest.xml, line 50459)
```

https://github.com/hashgraph/hedera-services/actions/runs/5247835607/jobs/9499523217?pr=7065

### PR DESCRIPTION
**Pull request by imalygin**
**Description**:

`MemoryIndexDiskKeyValueStoreMergeHammerTest` causes excessive log output due to its nature. Normally we have `DEBUG` level configured for the tests but for this test it's `WARN` now.

**Related issue(s)**:

Fixes #7083  

- [x] Documented

### CODE REVIEW

**Review by OlegMazurov — DISMISSED** [review]

File: platform-sdk/swirlds-jasperdb/build.gradle.kts [hunk 1]
```diff
@@ -15,26 +15,27 @@
...
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.library")
-  id("com.swirlds.platform.maven-publish")
-  id("com.swirlds.platform.benchmark-conventions")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.library")
+    id("com.swirlds.platform.maven-publish")
+    id("com.swirlds.platform.benchmark-conventions")
 }
```
Comment (cody-littley) [inline platform-sdk/swirlds-jasperdb/build.gradle.kts:22]: Are these formatting changes to this file intentional?

File: platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/MemoryIndexDiskKeyValueStoreMergeHammerTest.java [hunk 2]
```diff
@@ -47,11 +55,28 @@
...
     /** Temporary directory provided by JUnit */
     @SuppressWarnings("unused")
     @TempDir
     Path testDirectory;
 
+    @BeforeAll
+    public static void setup() {
+        LoggerContext ctx = (LoggerContext) LogManager.getContext(false);
+        Configuration config = ctx.getConfiguration();
+        LoggerConfig loggerConfig = config.getLoggerConfig(LogManager.ROOT_LOGGER_NAME);
```
Comment (cody-littley) [inline platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/MemoryIndexDiskKeyValueStoreMergeHammerTest.java:68]:
```suggestion
        final LoggerContext ctx = (LoggerContext) LogManager.getContext(false);
        final Configuration config = ctx.getConfiguration();
        final LoggerConfig loggerConfig = config.getLoggerConfig(LogManager.ROOT_LOGGER_NAME);
```

**Review by cody-littley — APPROVED** [review]

File: platform-sdk/swirlds-jasperdb/build.gradle.kts (hunk 1 above)
Comment (imalygin) [reply platform-sdk/swirlds-jasperdb/build.gradle.kts:22]: I thought it's done by Spotless. Let me revert it and see if the build fails

Comment (imalygin) [reply platform-sdk/swirlds-jasperdb/build.gradle.kts]: yep. it was spotless

**Review by nathanklick — APPROVED** [review]

**Review by cody-littley — APPROVED** [review]

**Review by OlegMazurov — APPROVED** [review]

**Pull request merged**